pkg/converter/constant.go

```diff
@@ -13,6 +13,9 @@ const (
 
 	ManifestNydusCache = "containerd.io/snapshot/nydus-cache"
 
+	ConfigGCLabelKey = "containerd.io/gc.ref.content.config"
```

Comment:
Seems it's a private constant, maybe put it into `convert_unix.go` and name `configGCLabelKey`?